components/setting_picture.jsx

```diff
@@ -180,17 +200,31 @@ export default class SettingPicture extends Component {
             );
         }
 
+        let helpText;
+        if (imageContext === 'team') {
+            helpText = (
+                <FormattedHTMLMessage
+                    id={'setting_picture.help.team'}
+                    defaultMessage='Upload a team icon in BMP, JPG or PNG format.<br>Images with a solid background color are recommended.'
```

Comment:
This is a design question for PMs, but couldn't we just fill in the background of transparent images with white like we do for profile pictures?

@esethna

Reply:
Yes, that was part of this [ticket](https://mattermost.atlassian.net/browse/MM-10354)